core/invoker/src/main/scala/whisk/core/container/HttpUtils.scala

```diff
@@ -132,5 +132,6 @@ protected[core] class HttpUtils(
     private val connection = HttpClientBuilder
         .create
         .setDefaultRequestConfig(httpconfig)
+        .useSystemProperties()
```

Comment:
There are few test cases which depends on this utils and it requires proxy configuration.